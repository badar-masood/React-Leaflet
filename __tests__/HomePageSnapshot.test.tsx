import renderWithProviders from "../jest.setup";
import HomePage from "../src/pages/home";

it("renders homepage unchanged", () => {
  const { container } = renderWithProviders(<HomePage />);
  expect(container).toMatchSnapshot();
});
