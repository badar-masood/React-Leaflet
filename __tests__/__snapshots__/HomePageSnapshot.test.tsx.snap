// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="chakra-input__group css-4302v8"
  >
    <div
      class="chakra-input__left-element css-1u2k0bl"
    >
      <svg
        class="chakra-icon css-1fdnb51"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M23.384,21.619,16.855,15.09a9.284,9.284,0,1,0-1.768,1.768l6.529,6.529a1.266,1.266,0,0,0,1.768,0A1.251,1.251,0,0,0,23.384,21.619ZM2.75,9.5a6.75,6.75,0,1,1,6.75,6.75A6.758,6.758,0,0,1,2.75,9.5Z"
          fill="currentColor"
        />
      </svg>
    </div>
    <input
      class="chakra-input css-h82jag"
      placeholder="Search..."
    />
  </div>
  <ul
    class="css-izdj8j"
  />
</div>
`;
